src/State/helpers.js

```diff
@@ -1,17 +1,34 @@
 // ideally, would like to have methods that all ducks will need here
 
 // takes in requests data from api results and returns parseable data
-export const transformRequests = requests => {
+export const transformRequests = (requests, role) => {
   return requests.map(request => {
     const reqData = JSON.parse(request.requestdata);
-    return {
-      ...request,
-      databasename: request.databasename || '', // placeholder for name property
-      requestdata: reqData,
-      description: reqData.description || '',
-      requeststatus:
-        request.requeststatus.charAt(0).toUpperCase() +
-        request.requeststatus.slice(1)
-    };
+    switch (role) {
```

Comment:
This was the only thing I can see getting an error when clicking on the request sent listItem.
![Screen Shot 2019-12-06 at 11 22 10 AM](https://user-images.githubusercontent.com/57810039/70350346-74253300-181b-11ea-927c-0b7d68a1f829.png)

Reply:
Trying to reproduce the error but I'm not running into it... can you tell me what you did to get here? @robertwollny-pnmac

Reply:
fixed! try running it again on your machine